app/comparator.py

```
@@ -0,0 +1,134 @@
+            
+            if result.returncode != 0:
+                raise Exception(f"Script failed: {result.stderr}")
+            
+            # Get the output file
+            current_date = datetime.now().strftime('%Y-%m-%d')
+            diff_file = f"{self.updates_dir}/{repo_name}-{current_date}-open-api-diff.md"
+            
+            # Read the diff content if file exists
+            diff_content = ""
+            if os.path.exists(diff_file):
+                with open(diff_file, 'r') as f:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)